persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/EnsemblRepositoryImpl.java

```diff
@@ -61,6 +61,20 @@ private EnsemblCanonical findOneCanonicalByHugoSymbol(String hugoSymbol) {
         return ensemblCanonical;
     }
 
+    private EnsemblCanonical findOneCanonicalByEntrezGeneId(String entrezGeneId) {
+        EnsemblCanonical ensemblCanonical;
+        Query query;
+
+        // search by entrez gene id
+        Criteria criteria = Criteria.where("entrez_gene_id").is(Integer.valueOf(entrezGeneId));
```

Comment:
seems like when using mongoimport, if values are postfixed with `.0` they are stored as floats. You can however still search by integers